samples/event_manager_proxy/README.rst

```diff
@@ -74,6 +74,12 @@ For example, you can do this from the command line using west by completing the
       cd build/event_manager_proxy_remote-prefix/src/event_manager_proxy_remote-build/
       west flash
 
+ To provide the ICMSG backend configuration, specify the ``-DCONF_FILE=prj_icmsg.conf`` parameter along with the build command:
+
+  .. code-bloc:: console
+
+     west build -p -b nrf5340dk_nrf5340_cpuapp -- -DCONF_FILE=prj_icmsg.conf
+
```

Comment:
```suggestion
ICMSG backend configuration
 To provide the ICMSG backend configuration, specify the ``-DCONF_FILE=prj_icmsg.conf`` parameter along with the build command:

  .. code-bloc:: console

     west build -p -b nrf5340dk_nrf5340_cpuapp -- -DCONF_FILE=prj_icmsg.conf

```
One concern here: this is the only place where we mention ICMSG configuration. For this reason, I would rather see these lines in a "Configuration" section with additional information about the ICMSG configuration, even just one line of context.